src/hoc/RealtimeTemplates.tsx

```diff
@@ -108,7 +101,7 @@ export const RealTimeTemplates = ({
           event: '*',
           schema: 'public',
           table: 'TaskTemplates',
-          filter: userRole === AssigneeType.internalUser ? `workspaceId=eq.${tokenPayload?.workspaceId}` : undefined,
+          filter: `workspaceId=eq.${tokenPayload?.workspaceId}`,
```

Comment:
I think we should wait for tokenPayload to be available before initializing a realtime channel

In cases when tokenPayload has not been resolved and put in redux, we would be opening a realtime channel with filter `workspaceId=eq.undefined`

Reply:
since tokenPayload is coming directly from server components and not the store, I think theres a guarantee of token payload working when if the token payload is valid.